LICENSE

```diff
@@ -307,11 +286,6 @@ For details see http://jquery.org/license
 
 ++++++++++++++++++++++++++++++
 
-Apache Trafodion uses asciidocs to create its documentation which is licensed
-under MIT 
-
-For details see http://asciidoctor.org/
```

Comment:
No, I don't think so. The docs do not include any ASCIIdoc software. It is merely used as a build tool to generate other formats (html, pdf) from asciidoc formatted files.